app/enterprise/0.32-x/getting-started/accessing-your-license.md

```diff
@@ -1,33 +1,42 @@
 ---
-title: How to Access Your Kong Enterprise Edition License
+title: Accessing Your Kong License File
 ---
+> If you are a Free Trial User please skip to the "Free Trial Users" section below
 
-Starting with Kong EE v .29, Kong requires a license file to start. This guide will walk you through accessing your license file. 
+# Accessing your License File
 
-Log into https://bintray.com/login?forwardedFrom=%2Fkong%2F
-If you are unaware of your login credentials, reach out to your CSE and they'll be able to assist you.
+Your license file can be found along side your Kong EE installation package in Bintray. To access Bintray, login here:
```

Comment:
"along side" -> "alongside" 

And, is it really physically to the side of Kong EE installation package in Bintray's UI? If not, perhaps make this less specific.